.env.example

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql+psycopg://chattoner-user:chattoner1234U*@127.0.0.1:5433/chattoner-db"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)